app/src/main/kotlin/dev/patrickgold/florisboard/app/settings/localization/LocalizationScreen.kt

```diff
@@ -118,17 +128,50 @@ fun LocalizationScreen() = FlorisScreen {
                             DisplayLanguageNamesIn.NATIVE_LOCALE -> subtype.primaryLocale.displayName(subtype.primaryLocale)
                         },
                         summary = summary,
-                        onClick = {
-                            navController.navigate(
-                                Routes.Settings.SubtypeEdit(subtype.id)
-                            )
-                        },
+                        modifier = Modifier.combinedClickable(
+                            onClick = {
+                                navController.navigate(
+                                    Routes.Settings.SubtypeEdit(subtype.id)
+                                )
+                            },
+                            onLongClick = {
+                                chosenSubtypeToDelete = subtype
+                            }
+                        )
                     )
+
                 }
             }
         }
-
-        //PreferenceGroup(title = stringRes(R.string.settings__localization__group_layouts__label)) {
-        //}
     }
+
+    DeleteSubtypeConfirmationDialog(
+        subtypeToDelete = chosenSubtypeToDelete,
+        onDismiss = {
+            chosenSubtypeToDelete = null
+        },
+        onConfirm = {
+            chosenSubtypeToDelete?.let { subtypeManager.removeSubtype(subtypeToRemove = it) }
+            chosenSubtypeToDelete = null
+        }
```

Comment:
Just a little formatting nit-pick :)
```suggestion
        },
```